packages/analytics-js/src/services/StoreManager/storages/CookieStorage.ts

```diff
@@ -33,20 +33,15 @@ class CookieStorage implements IStorage {
 
   configure(options: Partial<ICookieStorageOptions>): ICookieStorageOptions {
     this.options = mergeRight(this.options ?? {}, options);
-    this.isSupportAvailable = isStorageAvailable('cookieStorage', this);
+    this.isSupportAvailable = isStorageAvailable('cookieStorage', this, this.logger);
     this.isEnabled = Boolean(this.options.enabled && this.isSupportAvailable);
     return this.options;
   }
 
   setItem(key: string, value: Nullable<string>): boolean {
-    try {
-      cookie(key, value, this.options);
-      this.length = Object.keys(cookie()).length;
-      return true;
-    } catch (err) {
-      this.logger?.error(err, 'CookieStorage');
-      return false;
-    }
```

Comment:
why we removed the try catch?

Reply:
It is not required anymore. As explained in the below comments.